src/metpy/plots/declarative.py

```diff
@@ -754,6 +759,30 @@ def draw(self):
             self.ax.set_title(title)
             self._need_redraw = False
 
+    def __copy__(self):
+        """Return a copy of this MapPanel."""
+        # Create new, blank instance of MapPanel
+        cls = self.__class__
+        obj = cls.__new__(cls)
+
+        # Copy each attribute from current MapPanel to new MapPanel
+        for name in self.trait_names():
+            # The 'plots' attribute is a list.
+            # A copy must be made for each plot in the list.
+            if name == 'plots':
+                obj.plots = []
+                for plot in self.plots:
+                    obj.plots = obj.plots + [copy.copy(plot)]
+            else:
+                setattr(obj, name, getattr(self, name))
+
+        return obj
+
+    def copy(self):
+        """Return a copy of the panel."""
+        copied_obj = copy.copy(self)
+        return copied_obj
```

Comment:
Make this like the others?
```suggestion
        return copy.copy(self)
```